foo_024_051_vegetation_health_products/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3-stretch
+FROM python:3.7
```

Comment:
@weiqi-tori any reason not to upgrade to 3.9 here?

Reply:
Some other packages may be out of date and 3.9 cannot work. I didn't spend a lot of time testing the dockerfile since 3-stretch and 3.7 work fine.